tests/test_circuit_ids.py

```diff
@@ -209,6 +209,16 @@ def test_append(self):
         )
         assert test_obj == expected
 
+    def test_intersection(self):
+        first_ids = self.ids_cls.from_tuples([("a", 1), ("a", 2), ("a", 1), ("a", 2), ("b", 1)])
+        to_intersect = self.ids_cls.from_tuples([("b", 1), ("a", 3), ("a", 2)])
+        expected = self.ids_cls.from_tuples([("a", 2), ("b", 1)])
+        print(first_ids, first_ids.intersection(to_intersect))
```

Comment:
The print statement is probably a leftover and it should be removed.

Reply:
agreed -> removing